doc/requirements.txt

```diff
@@ -5,6 +5,11 @@ graphviz
 pybind11
 sphinx
 sphinx-automodapi
```

Comment:
Just a minor question. Do we want to pin sphinx to be `5.0`+? The error found in my readthedocs CI is `sphinx.errors.VersionRequirementError: 5.`0

Reply:
No, we don't. At least not for now.